### PR TITLE
Update striptags.rst

### PR DESCRIPTION
**Pull request by peter17**
Add information about supported second param.

### CODE REVIEW

File: doc/filters/striptags.rst [hunk 1]
```diff
@@ -13,3 +13,11 @@ by one space:
     Internally, Twig uses the PHP `strip_tags`_ function.
 
 .. _`strip_tags`: http://php.net/strip_tags
+
+    As a consequence, a second parameter, `$allowable_tags`, can be provided. In this example, the tags like `<br/>`, `<br>`, `<p>` and `</p>` will remain in the string:
```
Comment (stof) [inline doc/filters/striptags.rst]: this should be above the list of links (which should stay at the bottom of the file)

File: doc/filters/striptags.rst [hunk 2]
```diff
@@ -13,3 +13,11 @@ by one space:
...
+
+.. code-block:: jinja
+
+    {{ some_html|striptags('<br><p>') }}
+
+.. note::
```
Comment (stof) [inline doc/filters/striptags.rst]: why is there an empty note here

File: doc/filters/striptags.rst (hunk 1 above)
Comment (stof) [inline doc/filters/striptags.rst]: I would not say ``a second parameter`` but ``a parameter``, as the first argument of a filter does not look like a parameter in the Twig syntax

Comment (stof) [inline doc/filters/striptags.rst]: inline code blocks must used 2 backticks, not 1. This is a rST file, not a markdown one. A single backtick creates a rST reference

**Comment by peter17** [comment]
Replaced by #2371

**Pull request closed without merging**